app/mailers/application_mailer.rb

```diff
@@ -0,0 +1,4 @@
+# base mailer class
+class ApplicationMailer < ActionMailer::Base
+  default from: 'rmacg89+ukip@gmail.com'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.